emcc.py

```diff
@@ -1258,6 +1259,7 @@ def get_bitcode_file(input_file):
         if options.debug_level == 4 or not (final_suffix in JS_CONTAINING_SUFFIXES and options.js_opts):
           newargs.append('-g') # preserve LLVM debug info
           options.debug_level = 4
+          shared.Settings.DEBUG_LEVEL = 4
```

Comment:
this looks correct, but i'd make sure nothing breaks in the tests...

Reply:
Yeah, the previous behavior was that Settings.DEBUG_LEVEL was out of sync with options.debug_level so if anything depends on that it's surely a bug.